src/llmq/quorums_dkgsession.cpp

```diff
@@ -383,6 +385,9 @@ void CDKGSession::VerifyPendingContributions()
         memberIndexes.emplace_back(idx);
         vvecs.emplace_back(receivedVvecs[idx]);
         skContributions.emplace_back(receivedSkContributions[idx]);
+        // Note: Write here already to definitely store one contribution for each member no matter if
+        // our share is valid or not, "could" be that others are still correct?
```

Comment:
See 8db3ef614a371836289a0cd69ea97f212705342f

Just simplifies the language/wording of the comment